tools/add-badges.py

```diff
@@ -0,0 +1,71 @@
+import re
+import os
+import subprocess
+import fileinput
+import sys
+import shutil
+
+for repo_list_line in open('repositories.txt'):
```

Comment:
Can be `repositories.txt` obtained with https://github.com/OCA/maintainer-tools/blob/master/tools/oca_projects.py? If so, document it here.